content/tutorials/coordination.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Coordination Planner"
```

Comment:
Can we change this to be a bit more descriptive? Nothing in here mentions multiple mushr cars so people won't know what this is